sonarts-core/tslint-sonarts.json

```diff
@@ -1,7 +1,7 @@
 {
   "rules": {
     "cognitive-complexity": true,
-    "default-clauses-should-be-last": true,
+    "prefer-default-last": true,
```

Comment:
@andrea-guarino-sonarsource it's different alphabetical order now, you need to move it everywhere (sorry:))